test/new/db/cmd/cmd_macros

```diff
@@ -130,6 +130,24 @@ CMDS=<<EOF
 f foo.one = 1
 f foo.bar = 2
 f foo.cow = 3
-(,?v $$)() @@ foo*
+(;?v $$)() @@ foo*
```

Comment:
I will miss those anonymous macro calls with just the comma. But overall the change looks good to me. Will solve some issues for assembling and such